src/content/ui-api/web/organizations/access-control/index.md

```diff
@@ -0,0 +1,175 @@
+---
+linkTitle: Access control
+title: Access control for organizations in the web user interface
+description: Learn how to use our web UI for organization access control to quickly associate users, groups, and service accounts with organizations.
```

Comment:
'organization access control' is a bit of a mouthful - maybe something like 'for organization management'?

Reply:
Going for "organization RBAC", since "organization management" is more the broader context here.